commercetools-internal-docs/src/main/java/io/sphere/sdk/meta/ReleaseNotes.java

```diff
@@ -133,6 +133,13 @@
  <li class=fixed-in-release></li>
  </ul>
  -->
+ <h3 class=released-version id="v1_25_0">1.26.0 (19.10.2017)</h3>
```

Comment:
I like your idea to add release notes on the branch 👍 But we should put the correct version into the id attribute `id="v1_26_0"` 😉

Reply:
😅 yeah this is a stupid mistake